### PR TITLE
Change version suffix of System.Reflection.Metadata package to -rc3 for v1.3

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@nguerrera @ericstj

**Comment by ericstj** [comment]
Just delete it and inherit the default value of the repo.

File: src/System.Reflection.Metadata/pkg/future/System.Reflection.Metadata.pkgproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <!-- we need to be supported on pre-nuget-3 platforms (Dev12, Dev11, etc) -->
     <MinClientVersion>2.8.6</MinClientVersion>
     <!-- This is the .pkgproj for the "future" build which is not on the same release cadence as corefx -->
```
Comment (ericstj) [inline src/System.Reflection.Metadata/pkg/future/System.Reflection.Metadata.pkgproj]: I guess this comment isn't true??

**Comment by tmat** [comment]
What is 
<NuspecSuffix>-future</NuspecSuffix>
?

**Comment by nguerrera** [comment]
@tmat It's so that the nuspec file has a unique name and does not binclash with the 1.2 file. It is not observable to package end-user.

File: src/System.Reflection.Metadata/pkg/future/System.Reflection.Metadata.pkgproj (hunk 1 above)
Comment (tmat) [inline src/System.Reflection.Metadata/pkg/future/System.Reflection.Metadata.pkgproj]: It's kind of true - the cadence is more frequent than CoreFx.

Comment (tmat) [inline src/System.Reflection.Metadata/pkg/future/System.Reflection.Metadata.pkgproj]: I'll remove that comment. Doesn't add much value.

**Comment by nguerrera** [comment]
:+1:

**Comment by tmat** [comment]
@dotnet-bot retest Innerloop Windows_NT Debug Build and Test please
// Previous failure: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/windows_nt_debug_prtest/3118/
// Retest reason:

**Pull request merged**